Utils/hdinsight-node-common/src/com/microsoft/azure/projectarcadia/common/ArcadiaSparkCompute.java

```diff
@@ -104,7 +104,14 @@ public String getConnectionUrl() {
         }
 
         try {
-            return new URIBuilder(this.workSpace.getSparkUrl()).setPort(ARCADIA_SPARK_SERVICE_PORT).build().toString();
+//            return new URIBuilder(this.workSpace.getSparkUrl()).setPort(ARCADIA_SPARK_SERVICE_PORT).build().toString();
```

Comment:
>  return new URIBui [](start = 12, length = 19)

remove this line

Reply:
Let's keep this for what it should be. I'll add comments here.